src/posts/2021-09-06-using-css-frameworks.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "CSS Frameworks: Tailwind- en Windi CSS"
+category: inspiration
+tags: framework, css, sass, frontend
+comments: true
+author: nick
+min_read: 5
+date: 2021-09-06
+permalink: /2021/09/06/using-css-frameworks/
+---
+
+<!-- Section: Intro -->
+
+Veel websites, webwinkels en blogs maken gebruik van een CSS framework. Erg populair is nog altijd Twitter Bootstrap. Een ervaren webdesigner / frontender herkent het direct zonder in de code te hoeven kijken. Grafisch gezien lijken ze namelijk vaak sterk op elkaar. Bootstrap is snel inzetbaar en je hebt in no-time een responsive website staan. Zelf de source downloaden en Bootstrap builden wordt helaas nog vaak overgeslagen. Men plakt de productie CDN link in de head en de overrides worden er aan toegevoegd. Er kleven hier echter een aantal nadelen aan. Vaak wordt er maar een klein deel van de hele library gebruikt, terwijl de browser de volledige stylesheet in moet laden. "Eénmaal in je cache, geladen met 5G maakt toch niet uit?" zullen sommigen zeggen. Toch gaat dit ten kosten van de performance en de UX; verspringende fontsizes en kleuren zie ik helaas nog te vaak.
```

Comment:
>"Eénmaal in je cache, geladen met 5G maakt toch niet uit?" zullen sommigen zeggen.

Not quite sure how to read this sentence, but that might just be me (not required to change).

Reply:
It's a strange sentence, I will change